account_invoice_supplier_ref_unique/views/account_move.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <field name="ref" position="before">
```

Comment:
![Now](https://user-images.githubusercontent.com/31690536/111862771-afd40580-8957-11eb-9e2b-9812b70ceaf0.png)

`<xpath expr="//label[@for='ref']" position="before">`


![Captura de pantalla de 2021-03-20 08-35-31](https://user-images.githubusercontent.com/31690536/111862775-b82c4080-8957-11eb-85da-af50cf7b56c6.png)


Could you change please?

Reply:
Done